.circleci/config.yml

```
@@ -0,0 +1,32 @@
+version: 2
+jobs:
+  build-and-test:
+    docker:
+      - image: uxpin2/node8:1.0
+        auth:
+          username: $DOCKER_USER
+          password: $DOCKER_PASS
+    working_directory: ~/project/
+    steps:
+      - checkout
+      - run:
+          name: 'Login to NPM'
+          command: npm i -g npm-cli-login && npm-cli-login
+      - run:
+          name: "Install dependencies"
+          command: npm install
+      - run:
+          name: "Run tests"
+          command: npm run test
+      - run:
+          name: "Check ESLint"
+          command: ./node_modules/eslint/bin/eslint.js --ext js,jsx src --ignore-pattern *.tests.js
+      - run:
+          name: 'Push to UXPin'
+          command: ./node_modules/@uxpin/merge-cli/bin/uxpin-merge push --webpack-config ./webpack.config.js --wrapper ./src/UXPinWrapper/UXPinWrapper.js --uxpin-domain uxpin.com --token $TOKEN
```

```suggestion
          command: ./node_modules/.bin/uxpin-merge push --webpack-config ./webpack.config.js --wrapper ./src/UXPinWrapper/UXPinWrapper.js
```

- `--token` is not needed as we read from `UXPIN_AUTH_TOKEN` Env variable by default
- `--uxpin-domain` is not needed as `uxpin.com` is a default value